spec/rubocop/cop/style/method_called_on_do_end_block_spec.rb

```diff
@@ -5,62 +5,57 @@
 
   context 'with a multi-line do..end block' do
     it 'registers an offense for a chained call' do
-      inspect_source(cop, <<-END.strip_indent)
+      expect_offense(<<-RUBY.strip_indent)
         a do
           b
         end.c
-      END
-      expect(cop.offenses.size).to eq(1)
-      expect(cop.highlights).to eq(['end.c'])
+        ^^^^^ Avoid chaining a method call on a do...end block.
```

Comment:
Or do you just mean when there are multiple lines? If that is what you mean I'd have to think for a bit. The rule is pretty simple: the carets only refer to the line above it. Not sure how it could be simpler. It also basically matches the RuboCop highlighting output which is why i figured it was fairly intuitive